pkg/cloudprovider/aws/object_store.go

```diff
@@ -207,5 +212,10 @@ func (o *objectStore) CreateSignedURL(bucket, key string, ttl time.Duration) (st
 		Key:    aws.String(key),
 	})
 
+	if o.signatureVersion == "1" {
```

Comment:
I think we may want to check for valid vs invalid values. If the user specifies anything other than 1, they get 4.

Reply:
Like this https://github.com/heptio/ark/pull/998/commits/2cbe34f3af58e41be653662cfca55f038f148ced ?